test/inductor/test_aot_inductor_custom_ops.py

```diff
@@ -0,0 +1,408 @@
+# Owner(s): ["module: inductor"]
+# This test requires libaoti_custom_ops.so to be built, which happnes when BUILD_TEST = 1
+import copy
+import logging
+import os
+import sys
+import types
+import unittest
+
+import torch
+import torch._export
+import torch._inductor
+import torch._inductor.config
+from torch._dynamo.testing import same
+from torch._inductor import config
+from torch._inductor.test_case import TestCase
+from torch.export import Dim, export
+from torch.testing import FileCheck
+from torch.testing._internal import common_utils
+from torch.testing._internal.common_utils import (
+    find_library_location,
+    IS_CI,
+    IS_FBCODE,
+    IS_MACOS,
+    IS_SANDCASTLE,
+    IS_WINDOWS,
+)
+from torch.testing._internal.logging_utils import LoggingTestCase, make_logging_test
+from torch.testing._internal.triton_utils import HAS_CUDA
+
+
+if IS_WINDOWS and IS_CI:
+    sys.stderr.write(
+        "Windows CI does not have necessary dependencies for test_torchinductor yet\n"
+    )
+    if __name__ == "__main__":
+        sys.exit(0)
+    raise unittest.SkipTest("requires sympy/functorch/filelock")
+
+try:
+    try:
+        from .test_aot_inductor_utils import AOTIRunnerUtil
+        from .test_torchinductor import copy_tests, TestFailure
+    except ImportError:
+        from test_aot_inductor_utils import (  # @manual=fbcode//caffe2/test/inductor:aot_inductor_utils-library
+            AOTIRunnerUtil,
+        )
+        from test_torchinductor import (  # @manual=fbcode//caffe2/test/inductor:test_inductor-library
+            copy_tests,
+            TestFailure,
+        )
+except (unittest.SkipTest, ImportError) as e:
+    if __name__ == "__main__":
+        sys.exit(0)
+    raise
+
+
+def check_model(
```

Comment:
Better to import these util functions from test_aot_inductor.py or a common util file instead of copying.